docs/spec/src/integration/spec/6-secure-integration.md

```diff
@@ -5,11 +5,13 @@ This page is meant to be read by eigenda and rollup developers who are writing a
 ## Validity Conditions
 
 EigenDA is a service that assures the availability and integrity of payloads posted to it for 14 days.
-When deriving a rollup chain by running its derivation pipeline, only EigenDA certs that satisfy 3 validity conditions are considered valid and used:
-1. RBN Recency Validation - ensure that the cert's reference block number (RBN) is not too old with respect to the L1 block at which the cert was included in the rollup's batcher-inbox. This ensures that the blob on EigenDA has sufficient availability time left (out of the 14 day period) in order to be downloadable if needed during a rollup fault proof window.
+When deriving a rollup chain by running its derivation pipeline, only EigenDA `DACerts` that satisfy the three validity conditions are considered valid and used:
+1. RBN Recency Validation - ensure that the DA Cert's reference block number (RBN) is not too old with respect to the L1 block at which the cert was included in the rollup's batcher-inbox. This ensures that the blob on EigenDA has sufficient availability time left (out of the 14 day period) in order to be downloadable if needed during a rollup fault proof window.
 2. Cert Validation - ensures sufficient operator stake has signed to make the blob available, for all specified quorums. The stake is obtained onchain at a given reference block number (RBN) specified inside the cert.
 3. Blob Validation - ensures that the blob used is consistent with the KZG commitment inside the Cert.
 
+If a certificate fails validation then it is disregarded from the rollup's derivation pipeline and treated as an empty entry.
```

Comment:
```suggestion
If a certificate fails validation then it MUST be discarded from the rollup's derivation pipeline.
```

Reply:
actually blob validation failing shouldnt lead to discarding of cert right...? cc @bxue-l2 
Because that just means the relay or validators are being malicious, but the cert itself is correct. it just points to another blob which we need to wait for (this should stall the derivation pipeline I think?)

Reply:
feel free to own @bxue-l2